README.md

```diff
@@ -15,11 +13,68 @@ Node FHIR client library
 
 # Example
 
-[See examples](./examples/index.js)
+Examples using promises...
 
-# Contributing
+```javascript
+const Client = require('../lib/client');
+const fhirClient = new Client({ baseUrl: 'https://sb-fhir-stu3.smarthealthit.org/smartstu3/open' });
 
-All are welcome to contribute. By participating in this project you agree to follow the [Code of Conduct](https://github.com/Vermonster/node-fhir-client/blob/master/CODE_OF_CONDUCT.md).
+fhirClient.smartAuthMetadata().then((response) => {
+  console.log(response);
+});
+
+fhirClient
+  .read({ resourceType: 'Patient', identifier: '2e27c71e-30c8-4ceb-8c1c-5641e066c0a4' })
+  .then((response) => {
+    console.log(response);
+  });
+
+fhirClient
+  .vread({ resourceType: 'Patient', identifier: '2e27c71e-30c8-4ceb-8c1c-5641e066c0a4', version: '1' })
```

Comment:
should this particular identifier be declared once, since it repeats?

or even: "constantized" ?  (by js convention, as `ALL_CAPS`, e.g.: `PATIENT_IDENTIFIER`)

Reply:
OTOH, these are supposed to be independent examples.